cli/src/alluxio.org/cli/cmd/exec/test_journal_crash.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package exec
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+)
+
+var TestJournalCrash = &TestJournalCrashCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "journalCrashTest",
+		JavaClassName: "alluxio.cli.JournalCrashTest",
+		ShellJavaOpts: fmt.Sprintf(env.JavaOptFormat, env.ConfAlluxioLoggerType, "Console"),
+	},
+}
+
+type TestJournalCrashCommand struct {
+	*env.BaseJavaCommand
+	creates   int
+	deletes   int
+	maxAlive  int
+	renames   int
+	testDir   string
+	totalTime int
+}
+
+func (c *TestJournalCrashCommand) Base() *env.BaseJavaCommand {
+	return c.BaseJavaCommand
+}
+
+func (c *TestJournalCrashCommand) ToCommand() *cobra.Command {
+	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
+		Use:   "journalCrashTest",
+		Short: "Test the Master Journal System in a crash scenario.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.Run(args)
+		},
+	})
+	cmd.Flags().IntVar(&c.creates, "creates", 2,
```

Comment:
i assume the java code has checks to ensure the integer values are valid -> threads must be positive numbers. these types of input checks should eventually shift to the go code. not necessary to do as part of this PR